index.d.ts

```diff
@@ -1,5 +1,11 @@
 import { Application, Context } from 'egg';
 import { MockMate } from 'mm';
+import { Test } from 'supertest';
+
+interface EggTest extends Test {
+  unexpectHeader(name: string, b: Function): EggTest;
```

Comment:
```suggestion
  unexpectHeader(name: string, b?: Function): EggTest;
```
`b` should be optional